packages/plugin-text/src/toolbar/button.tsx

```diff
@@ -1,35 +1,39 @@
-import { defaultTheming, styled, EditorTheming } from '@edtr-io/ui'
+import {
+  defaultTheming,
+  styled,
+  createUiElementTheme,
+  EditorThemeProps
+} from '@edtr-io/ui'
 
-export const ButtonGroup = styled.div({
-  position: 'absolute',
-  top: '-1em',
-  zIndex: 20
```

Comment:
This `zIndex` should still be set, otherwise the Overlay will be behind the ButtonGroup
![grafik](https://user-images.githubusercontent.com/11499926/55873815-ed875b80-5b90-11e9-8280-ad1a77809edd.png)

Reply:
I changed the z-Index of the inline-overlay